airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -80,6 +80,10 @@ class KubernetesPodOperator(BaseOperator):
         /airflow/xcom/return.json in the container will also be pushed to an
         XCom when the container completes.
     :type do_xcom_push: bool
+    :param is_delete_operator_pod: If True, delete the pod after it's reached
+        a final state (independent of pod success), or the task instance gets 
```

Comment:
Nit: Since this is a boolean param, can you change the description to "If... else" instead of "If ... or". Reading this, it's a bit confusing what this operator does.

Reply:
+1

Reply:
rephrased for clarity, as that is not my intended meaning!